src/components/Carousel.js

```diff
@@ -0,0 +1,59 @@
+import carousel from "../styles/carousel";
+import carouselItems from "../assets/carousel-slides/carouselItemsData";
+import { useState } from "react";
+
+import ForwardIcon from "@material-ui/icons/Forward";
+
+const carouselItemsCount = carouselItems.length - 1;
+
+const Carousel = () => {
+  const carouselStyles = carousel();
+  const [currentSlide, setCurrentSlide] = useState(0);
+
+  const showNextSlide = () => {
+    if (currentSlide < 2) {
+      setCurrentSlide(currentSlide + 1);
+    } else {
+      setCurrentSlide(0);
+    }
+  };
+
+  const showPreviousSlide = () => {
+    if (currentSlide > 0) {
+      setCurrentSlide(currentSlide - 1);
+    } else {
+      setCurrentSlide(carouselItemsCount);
+    }
+  };
+
+  return (
+    <div className={carouselStyles.carousel}>
+      <div
+        className={carouselStyles.carouselInner}
+        style={{ backgroundImage: `url(${carouselItems[currentSlide].img})` }}
+      >
+        <div className={carouselStyles.arrowContainer}>
+          <ForwardIcon
+            className={`${carouselStyles.arrowIcon} ${carouselStyles.backArrow}`}
+            style={{ fontSize: "5rem" }}
```

Comment:
and here, code is duplicated for 2 icons, but could be just `carouselArrowIcon` className

Reply:
I added inline styles initially because Material-UI icons get their styles from .MuiSvgIcon-root which is superior to other classes, so normal className won't work because they get overriden. A workaround is to use !important in CSS, so now changing arrowIcon class will apply to all of them. I've changed to !important 5 rem to have one class in CSS.